src/KeyHandler.cpp

```diff
@@ -169,6 +169,14 @@ bool KeyHandler::handle(Key key, McBopomofo::InputState* state,
         std::string value = choosingCandidate->candidates[0].value;
         auto committingState = std::make_unique<InputStates::Committing>(value);
         stateCallback(std::move(committingState));
+
+        if (associatedPhrasesEnabled_) {
+          std::unique_ptr<InputStates::AssociatedPhrasesPlain> newState =
+              buildAssociatedPhrasesPlainState(value);
+          if (newState != nullptr) {
+            stateCallback(std::move(newState));
```

Comment:
The `stateCallback` is no longer valid (it's now a const reference to a dangling object) because the commit state above will cause the candidate object (which owns the callback object) to be destoryed.

Please review https://github.com/openvanilla/fcitx5-mcbopomofo/pull/108; once it's merged, please merge with the mainline so that this will work safely.